lib/Fhp/Segment/Paginateable.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Fhp\Segment;
+
+/**
+ * Marks a request segment as supporting pagination, this means that the bank can split the result into several
+ * response segments that have to be queried one by one via the pagination token.
+ */
+interface Paginateable
+{
+    public function setPaginationToken(?string $paginationToken);
```

Comment:
Die Funktion wird zumindest in der Praxis nie mit null aufgerufen. Also könnte das `?` auch weg, außer es gibt einen Grund warum man das Token rauslöschen können sollte.